translations/components.tex/de.po

```diff
@@ -253,6 +379,26 @@ msgid ""
 "    \\caption{\\textbf{Adventure Card} \\textit{(Battlefield Expansion)}}\n"
 "  \\end{subfigure}"
 msgstr ""
+"      \\node at (0, 0) {\\includegraphics[width=0.8\\linewidth]{\\cards/event-back.png}};\n"
+"      \\node at \\iftoggle{printable}{(1, 0.5)}{(1, 1)} {\\includegraphics[width=0.8\\linewidth]{\\cards/event.png}};\n"
+"    \\end{tikzpicture}\n"
+"    \\caption{\\textbf{\\pagelink{Events}{Ereigniskarten}}\\\\\\textit{(Festung-Erw.)}}\n"
+"  \\end{subfigure}\n"
+"  \\begin{subfigure}[b]{0.25\\linewidth}\n"
+"    \\centering\n"
+"    \\includegraphics[width=0.8\\linewidth]{\\iftoggle{printable}{\\cards/astrolog-shorter.png}{\\cards/astrolog.png}}\n"
```

Comment:
```suggestion
"    \\iftoggle{printable}{\\includegraphics[width=0.7\\linewidth]{\\cards/astrolog.png}}{\\includegraphics[width=0.8\\linewidth]{\\cards/astrolog.png}}\n"
```

Reply:
This will do, please delete the image.

Reply:
The shorter image has different layout, not just smaller size - the overlap between front and back side is increased there.

It could be done via LaTeX, if the two pieces (front and back) are separate files, like done for the events cards. And I agree, this would be cleaner than having the combined images as PNG

Reply:
Eh, well ok

Reply:
In that case, please move the image to `translations/assets/de`

Reply:
Okay what exactly should i do here? ^^ i know that you both discuss about the Astrologercard Image but i'm not sure about the result of the discussion. 
Your, qwrtln, above suggestions for changing latex Code are not actual, right? Instead i should move the Image? But where Do I find the changed Image, which i should Move? And after moving is there something to change in the latex Code?

Reply:
@DarthGandalf Care to help?

Reply:
Done. This is temporary solution, I'd like to make it through latex at some point...